### PR TITLE
Create Profile Page

### PR DESCRIPTION
**Pull request by lattai**
The Image handling functions in gameHelper.js are not new, just refactored from characterDesign.js

### CODE REVIEW

File: src/main/webapp/profilePage.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+function fillText() {
+  getName();
+  getExperiencePoints();
+  getCareerPath();
+}
+
+function getName() {
+//   fetch("/name")
```
Comment (lzoebaker) [inline src/main/webapp/profilePage.js:19]: Change to "get-player-name", then un-comment to integrate with Many's (now-merged) code.

File: src/main/webapp/profilePage.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  const expContainer = document.getElementById('experience-container');
+  expContainer.innerText = "EXP:" + expString;
+  });
+}
+
+function getCareerPath() {
+  fetch("/game-dialogue")
+  .then(response => response.text())
+  .then(dialogueString => {
+  var dialogueArray = dialogueString.split("\n");
```
Comment (lzoebaker) [inline src/main/webapp/profilePage.js:41]: might need to indent?

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -512,3 +512,23 @@ audio {
   top:0;
   left:0;
 }
+
+/** classes for ProfilePage.html **/
+#gridAccessory {
+  object-fit: cover;
+  height: 150px;
+  width: 150px;
+}
```
Comment (lzoebaker) [inline src/main/webapp/style.css:521]: add spaces between these styling segments.

File: src/main/webapp/gameHelper.js [hunk 4]
```diff
@@ -65,6 +73,32 @@ function nextLine() {
   quoteContainer.innerText = dialogueArray[dialogueRegex];
 }
 
+//loads the image upload url to the form
```
Comment (lzoebaker) [inline src/main/webapp/gameHelper.js:76]: Add to your PR description that these functions aren't new, just refactored to gameHelper.js

File: src/main/webapp/profilePage.js (hunk 1 above)
Comment (menynexi) [reply src/main/webapp/profilePage.js:19]: +1 would pull master

File: src/main/webapp/gameHelper.js (hunk 4 above)
Comment (menynexi) [reply src/main/webapp/gameHelper.js:76]: there should be a space between // and load

File: src/main/webapp/gameHelper.js [hunk 5]
```diff
@@ -225,9 +259,61 @@ function reconstructWordToSentence(words) {
...
+      accessoryContainer = document.getElementById(HAT_IMAGE_ID);
+      break;
+    case COMPANION_STRING:
+      accessoryContainer = document.getElementById(COMPANION_IMAGE_ID);
+      break;
+    case GLASSES_STRING:
+      accessoryContainer = document.getElementById(GLASSES_IMAGE_ID);
+      break;
+  }
+//   if (accessoryContainer.src.equals())
```
Comment (menynexi) [inline src/main/webapp/gameHelper.js:299]: remove unused code of comment with a different format

File: src/main/webapp/profilePage.js (hunk 2 above)
Comment (menynexi) [reply src/main/webapp/profilePage.js:41]: +1

**Pull request merged**